xml/http_servers.xml

```diff
@@ -1766,9 +1766,10 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^PHP/(\S+)$">
-      <example>PHP/4.4.2-1build1</example>
-      <example>PHP/5.1.2</example>
+   <fingerprint pattern="^PHP/(\S+)(?:\s.+)?$">
+      <example service.component.version="4.4.2-1build1">PHP/4.4.2-1build1</example>
+      <example service.component.version="5.1.2">PHP/5.1.2</example>
+      <example service.component.version="5.3.14">PHP/5.3.14 ZendServer/5.0</example>
```

Comment:
IMO, if the only current example of a new "banner" that doesn't match what is already here only includes the ZendServer information, IMO it is worth adding a new fingerprint that explicitly matches PHP+ZendServer and also asserts ZendServer and the version.